src/ffi/cl_gl_h.rs

```diff
@@ -1,5 +1,35 @@
 //! OpenGL Extensions.
 
+use ffi::ClGlUint;
+use cl_h::{ cl_context, cl_mem_flags, cl_command_queue, cl_int,
+    cl_uint, cl_mem, cl_event };
+
+//#[link_args = "-L$OPENCL_LIB -lOpenCL"]
+#[cfg_attr(target_os = "macos", link(name = "OpenGL", kind = "framework"))]
+#[cfg_attr(target_os = "windows", link(name = "OpenGL"))]
+#[cfg_attr(not(target_os = "macos"), link(name = "OpenGL"))]
```

Comment:
I was wrong. It's not correct. Should be exactly like the original:

```
#[cfg_attr(target_os = "macos", link(name = "OpenCL", kind = "framework"))]
#[cfg_attr(target_os = "windows", link(name = "OpenCL"))]
#[cfg_attr(not(target_os = "macos"), link(name = "OpenCL"))]
```

It's still part of the OpenCL library, not OpenGL.